### DISCUSSION TITLE
Everything works except playback

### DISCUSSION THREAD

**Discussion by olivesoop** [jeffvli/feishin#302] [Q&A]
I've been using Feishin for a few weeks, but today music just isn't playing. I can play music through the web interface of Navidrome, I can play music through SonicXD. All of my music shows up with the correct information and metadata, but when I hit play nothing happens. Is there any sort of debug mode where I can see what's going on? I'm not a developer just a homelab tinkerer so not sure I'd even know what to look for there but figured it was worth asking.

**Comment by BlackDragon-B** [comment]
Try switching your audio backend to web in the settings and see if it works then.
Edit: You should've opened an issue btw, not a discussion.

**Reply by olivesoop** [reply]
That worked, thank you! And apologies, I don't use GitHub very often so not terribly familiar with what goes where.

**Reply by BlackDragon-B** [reply]
You are not in the clear yet, your MPV instance is not working correctly. Check if the executable is correct and try manually using mpv.
